R/pkg/R/generics.R

```diff
@@ -1022,6 +1059,7 @@ setGeneric("month", function(x) { standardGeneric("month") })
 #' @export
 setGeneric("months_between", function(y, x) { standardGeneric("months_between") })
 
+#' @param x a SparkDataFrame or a Column object.
 #' @rdname nrow
```

Comment:
here for "n" column function, it shouldn't be under rdname nrow, which is for count for DataFrame - I'd change this to a new rdname, `count` and put "count" and "n" under that.